src/PREfastXmlSarifConverter/SarifFormat.h

```diff
@@ -118,9 +118,9 @@ class SarifAnnotatedCodeLocation
         m_values[L"kind"] = kind;
     }
 
-    void SetEssential(bool essential)
+    void SetImportance(const std::wstring &importance)
     {
-        m_values[L"essential"] = json::Value(essential);
+		m_values[L"importance"] = json::Value(importance.c_str());
```

Comment:
we don't explicitly call .c_str() in other helpers, probably best to remain consistent.